os/fs/smartfs/smartfs_mksmartfs.c

```diff
@@ -83,7 +83,7 @@
  * Name: mksmartfs
  *
  * Description:
- *   Make a SMART Flash file system image on the specified block device
+ *   Make a jSmartFs Flash file system image on the specified block device
```

Comment:
while SMART Flash file system makes sense,
jSmartFs Flash file system does not.
So the sentence following the name would also need to be changed accordingly.